File: Java/Kot.java
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        String s = input.next();
        int i = 0, count = 0;
        while (i < s.length()) 
        {
            int kIndex = -1, oIndex = -1, tIndex = -1;
            for (int j = i; j < s.length(); j++) 
            {
                if (s.charAt(j) == 'k' && kIndex == -1)
                    kIndex = j;
                else if (s.charAt(j) == 'o' && kIndex != -1 && oIndex == -1)
                    oIndex = j;
                else if (s.charAt(j) == 't' && oIndex != -1 && tIndex == -1) 
                {
                    tIndex = j;
                    break;
                }
            }
            if (kIndex != -1 && oIndex != -1 && tIndex != -1) 
            {
                count++;
                i = tIndex + 1;
            } 
            else
                break;
        }
        System.out.print(count == 0 ? "NIE" : count);
        input.close();
    }
}

File: Python/Kot.py
i = 0
count = 0
s = input()
while i < len(s):
    k_index = -1
    o_index = -1
    t_index = -1
    for j in range(i, len(s)):
        if s[j] == 'k' and k_index == -1:
            k_index = j
        elif s[j] == 'o' and k_index != -1 and o_index == -1:
            o_index = j
        elif s[j] == 't' and o_index != -1 and t_index == -1:
            t_index = j
            break
    if k_index != -1 and o_index != -1 and t_index != -1:
        count += 1
        i = t_index + 1
    else:
        break
print('NIE' if count == 0 else count)